operations/availabilityblocks.md

```diff
@@ -258,7 +261,8 @@ Adds availability blocks which are used to group related [Availability updates](
 | `Name` | string | optional | The name of the block. |
 | `FirstTimeUnitStartUtc` | string | required | Start of the time interval, expressed as the timestamp for the start of the first time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
 | `LastTimeUnitStartUtc` | string | required | End of the time interval, expressed as the timestamp for the start of the last time unit, in UTC timezone ISO 8601 format. See [Time units](../concepts/time-units.md). |
-| `ReleasedUtc` | string | required | The moment when the block and its availability is released. |
+| `ReleasedUtc` | string | optional | The moment when the block and its availability is released, in UTC timezone ISO 8601 format. |
```

Comment:
Do we need to add the same statement about being mutually exclusive to the fields here like you did with `Get all availability`, or does it not apply in this case?

Reply:
Added: "Takes precedence over `RollingReleaseOffset`."